docs/source/getting-started/model-registry/index.rst

```diff
@@ -0,0 +1,25 @@
+Tutorial Overview
+=================
+
+The Model Registry is a centralized model store, set of APIs, and UI, to collaboratively manage the 
```

Comment:
Could we convert these long lists into 2 bullet point lists that explain what the components are and what features they bring to the table?